src/data/zwave_js.ts

```diff
@@ -151,12 +151,13 @@ export interface QRProvisioningInformation {
   maxInclusionRequestInterval?: number | undefined;
   uuid?: string | undefined;
   supportedProtocols?: Protocols[] | undefined;
+  [key: string]: any; // Allow for additional properties
```

Comment:
I think that we should not add this, and if we are to use a key that we know can exist, we add and type it. Otherwise using this object is going to suck, as all keys are valid.